arm/Microsoft.Network/virtualWans/readme.md

```diff
@@ -1,6 +1,6 @@
-# Virtual Wan `[Microsoft.Network/virtualWans]`
+# Virtual WANs `[Microsoft.Network/virtualWans]`
 
-This template deploys Virtual Wan
+This template deploys a Virtual WAN.
```

Comment:
I guess we should again use lower case for `virtual`

Reply:
done